modules/text/src/ocr_beamsearch_decoder.cpp

```diff
@@ -566,7 +566,7 @@ OCRBeamSearchClassifierCNN::OCRBeamSearchClassifierCNN (const string& filename)
 
     nr_feature = weights.rows;
     nr_class   = weights.cols;
-    patch_size  = (int)sqrt((float)kernels.cols);
+    patch_size  = (int)round(sqrt((float)kernels.cols));
```

Comment:
`cvRound()` is preferred for `float->int` conversion. `round()` is `float->float`.